### PR TITLE
Beats

### PR DESCRIPTION
**Pull request by yokoryu-meteor**
This PR adds the BEATs model

### CODE REVIEW

File: birdset/modules/models/LTU.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
+import torch
```
Comment (raphaelschwinger) [inline birdset/modules/models/LTU.py]: @yokoryu-meteor Please add LTU in a seperate PR.

File: configs/experiment/biofoundation/linear_classifier/audiomae_HSN.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# @package _global_
```
Comment (raphaelschwinger) [inline configs/experiment/biofoundation/linear_classifier/audiomae_HSN.yaml]: @yokoryu-meteor you based this branch on your audioMAE branch and not main, thats why you also have included the audiomae changes. Please remove them from this branch (rebase on main / revert the commits)

**Review by raphaelschwinger — COMMENTED** [review]
@yokoryu-meteor I moved the BEATs folder and fixed the imports. Also I applied a flattening of the embeddings to make it run. Unfortunately the produced embeddings contain `nan` values. Please debug to find out what is happening here.

**Comment by PariaValizadeh** [comment]
@raphaelschwinger Hi,  I made a commit for fixing NAN loss functio 2bf0ee8f9be3bf905c5b92691d61573c20406d02

File: configs/experiment/biofoundation/linear_classifier/audiomae_HSN.yaml (hunk 2 above)
Comment (raphaelschwinger) [reply configs/experiment/biofoundation/linear_classifier/audiomae_HSN.yaml]: I remvoed those changes.

**Comment by raphaelschwinger** [comment]
@PariaValizadeh Thanks for finding the bug!
